ports/stm32f3/boards.c

```diff
@@ -267,25 +267,7 @@ void OTG_FS_IRQHandler(void)
 
 #endif
 
-void USB_HP_IRQHandler(void)
```

Comment:
these IRQ should be remained, `OTG_FS_IRQHandler()` is the one should be removed.